src/onefetch/cli.rs

```diff
@@ -253,6 +254,11 @@ impl Cli {
             .takes_value(true)
             .help("Ignore all files & directories matching EXCLUDE."),
         )
+        .arg(
+            Arg::with_name("remote")
+            .long("remote")
+            .help("Clone repository from remote"),
+        )
```

Comment:
I think it would be useful if this arg either
- Had another arg that [required](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.requires) this arg
- Took a second value

So that the user could specify whether the cloned repo should be deleted or kept.

e.g. `onefetch --remote <my URL> --keep` to keep the repo after cloning it.

Reply:
> I think it would be useful if this arg either
> 
>     * Had another arg that [required](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.requires) this arg
> 
>     * Took a second value
> 
> 
> So that the user could specify whether the cloned repo should be deleted or kept.
> 
> e.g. `onefetch --remote <my URL> --keep` to keep the repo after cloning it.

Where would you like to keep this repository?

Reply:
You could probably just clone it to the current directory if it's being kept.

Reply:
> You could probably just clone it to the current directory if it's being kept.

I think it does not make much sense, because you could just do `git clone` and then run onefetch there.

Purpose of this feature is to quickly look up information about repository while not caring at all about the repository itself, but just about statistics.

Reply:
>Purpose of this feature is to quickly look up information about repository while not caring at all about the repository itself, but just about statistics.

Yes, that makes sense, but, since it is not doing a shallow clone, you're getting the whole repository anyway. For example, if this feature was used on [the Linux repo](https://github.com/torvalds/linux), it would take a long time. Worse if the user uses this feature and *then* decides to clone it to do some work on it. It is hardly "quickly looking up information."

For example, somebody could use this feature, and then decide it's a repository that they would want to work on based on the stats. If this feature doesn't support keeping the repository, they would have to clone it a second time.

Although I agree that it is unnecessary to add a `--keep` option *if* the repo is cloned with a shallow depth, since onefetch's clone would take less time, and a shallow clone isn't that useful for actual work. :slightly_smiling_face: